source/connect/connections.txt

```diff
@@ -0,0 +1,73 @@
+.. _connections-sidebar:
+
+===================
+Connections Sidebar
```

Comment:
Is it safe to call our only sidebar the connections sidebar? It also contains things that aren't connections and in future it will likely also contain streams, for example. Wondering if there's a better way to refer to the connections section in there.

Reply:
I guess this is all referring to just "connections list.

Reply:
I changed all mentions to just say "Compass sidebar"